roles/openshift_rhv/tasks/build_vm_list.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Creates a dictionary for use with oVirt.vm-infra role
+# https://github.com/oVirt/ovirt-ansible-vm-infra
+- name: Create virtual machine list fact
+  set_fact:
+    vms: >-
+      {{ vms|default([]) }} + [
+      {% for iter in range(item.count) -%}
+      {% if iter > 0 -%},{% endif -%}
+      {
+      'name': '{{ item.name }}{{ iter }}.{{ public_hosted_zone }}',
+      'tag': 'openshift_{{ item.profile }}',
+      'cloud_init':
+      {
+      'host_name': '{{ item.name }}{{ iter }}.{{ public_hosted_zone }}',
+      'authorized_ssh_keys': '{{ openshift_rhv_ssh_key }}'
```

Comment:
Can be added to the [all:vars]

Like:
[all:vars]
ansible_ssh_private_key_file = '~/.ssh/id_rsa.pub'